app/views/editions/_reviewer_field.html.erb

```diff
@@ -1,12 +1,18 @@
 <% error_class = @resource.errors.has_key?(:reviewer) ? ' error has-error' : '' -%>
```

Comment:
Do we know why this has both error and has-error? I'm guessing we only need has-error in this new world?

Reply:
Ideally, I'd like just one and `has-error` seems the best option as it'll be used by both with- and without-JS.  But, there are quite a few places to change, so I suggest a separate PR for this too.

Reply:
Have created a separate card for this story